### PR TITLE
Refactor package & class name

### PR DESCRIPTION
**Pull request by KeonHee**
* `connector` -> `explorer`
image

 * `navigation` -> `toolwindow`
 * `WhiskNavigator` -> `WhiskExplorerWindowForm`
 
 * Fix NPR for error log message

### CODE REVIEW

File: src/main/java/org/apache/openwhisk/intellij/wskdeploy/toolwindow/action/ChooseWskDeployBinAction.java [hunk 1]
```diff
@@ -49,7 +49,11 @@ public void actionPerformed(@NotNull AnActionEvent e) {
             NOTIFIER.notify(e.getProject(), file.getName() + " file has been registered with wskdeploy.", NotificationType.INFORMATION);
             EventUtils.publish(e.getProject(), ChooseWskDeployBinaryListener.TOPIC, l -> l.chooseWskDeployBinary(validWskDeploy.get()));
         } else {
-            NOTIFIER.notify(e.getProject(), file.getName() + " file cannot be used with wskdeploy.", NotificationType.ERROR);
+            if (file == null) {
+                NOTIFIER.notify(e.getProject(), "The file cannot be chosen", NotificationType.ERROR);
```
Comment (style95) [inline src/main/java/org/apache/openwhisk/intellij/wskdeploy/toolwindow/action/ChooseWskDeployBinAction.java:53]: In which case `file` becomes `null`?

File: src/test/java/org/apache/openwhisk/intellij/common/utils/whisk/WhiskUtilsTest.java [hunk 2]
```diff
@@ -26,7 +28,7 @@ public void getApihHostWithProtocol() {
         String expected1 = "https://foo.com";
         String expected2 = "http://foo.com";
 
-        assertEquals(expected1, WhiskUtils.getApihHostWithProtocol("foo.com"));
+        Assert.assertEquals(expected1, WhiskUtils.getApihHostWithProtocol("foo.com"));
```
Comment (style95) [inline src/test/java/org/apache/openwhisk/intellij/common/utils/whisk/WhiskUtilsTest.java:31]: Is this a required change?

File: src/main/java/org/apache/openwhisk/intellij/wskdeploy/toolwindow/action/ChooseWskDeployBinAction.java (hunk 1 above)
Comment (KeonHee) [reply src/main/java/org/apache/openwhisk/intellij/wskdeploy/toolwindow/action/ChooseWskDeployBinAction.java:53]: If the user doesn't select the file and close it, `file` will be `null`.

Comment (style95) [reply src/main/java/org/apache/openwhisk/intellij/wskdeploy/toolwindow/action/ChooseWskDeployBinAction.java:53]: I suppose in such a case, a user would just cancel the selection in most cases.
Do we really need to open a notification popup for it?

Comment (KeonHee) [reply src/main/java/org/apache/openwhisk/intellij/wskdeploy/toolwindow/action/ChooseWskDeployBinAction.java]: Good point. Fixed the action to return if the user doesn't select the file.

**Pull request merged**